frontend/src/pages/Activities/AddActivity/AddActivity.tsx

```diff
@@ -49,17 +50,27 @@ const AddActivity = ({}: AddActivityProps) => {
     const { data: autocompleteChoices } = useItemTypeAutoCompleteSuggestions(itemTypeSlug)
     const [popoverOpen, setPopoverOpen] = useState(false)
     const [form] = useForm()
+    const {data: item, status: itemStatus} = useItem(token)
+
     if (!itemType) {
         return <Spin />
     }
 
+    if (token && !item){
+        return <Spin/>
+     }
+
+
     return (
         <div>
             Add {itemType.name}
             <Form
                 initialValues={{
+                    ...item?.info??{},
+                    item__Parent: item?.parent_token??{},
```

Comment:
```suggestion
                    item__Parent: item?.parent_token
```

Reply:
this one, its fine if its undefined.  using the spread operator on undefined was weird; here {} isnt the right fallback anyway, this is a string